tools/checks/style/hn_check/src/hn_check.py

```diff
@@ -224,7 +235,7 @@ def get_identifier_prefix(identifier):
     indirection = "p" * (len(identifier) - len(direct_identifier))
     for key in TYPE_PREFIXES:
```

Comment:
Though this line didn't change, looks like we can just do `for prefix in TYPE_PREFIXES.values():`, `key` is only used to get values from this dictionary.